src/main/java/ttt/GameEngine.java

```diff
@@ -0,0 +1,65 @@
+package ttt;
+
+public class GameEngine {
+
+    public Board board;
+    public Player currentPlayer;
+    public Player nextPlayer;
+    private Player player1;
+    private Player player2;
+
+    public GameEngine(Player player1, Player player2, Board board) {
+        this.board = board;
+        this.currentPlayer = player1;
+        this.nextPlayer = player2;
+        this.player1 = player1;
+        this.player2 = player2;
+    }
+
+    public void play(int location) {
+        board.placeMark(currentPlayer.getMark(), location);
+        String status = board.getStatus();
+        if (status.equals("mark placed")) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2576103514/issues/source?bid=3284885&fileBranchId=3299578#l22)